### PR TITLE
added Wishlist and cart

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (825)
Screenshot (826)

### CODE REVIEW

File: library/Components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+:root {
+  --color-btn-cart: rgb(255, 69, 0);
+  --color-btn-bg-white: rgb(255, 255, 255);
+}
+
+.heading__container,
+.checkout__container {
+  text-align: center;
+  margin: 3% 0 2% 0;
```
Comment (ShrutiLakhani) [inline library/Components/cart/cart.css:9]: Haven't seen margin in %. You might have done it on purpose.

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean and crisp. Good job.

File: library/Components/cart/cart.css (hunk 1 above)
Comment (ashishkatiyar2404) [reply library/Components/cart/cart.css:9]: not on purpose will change to px.

**Comment by shruthipatil11** [comment]
Ui is great and code is clean and well structured except those margin in %.If its done, good to merge

File: library/Components/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    />
+    <script
+      src="https://kit.fontawesome.com/743732d519.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>E-Commerce</title>
+  </head>
+  <body>
+    <!-- NAV BAR START -->
+    <header>
```
Comment (m26unkwn) [inline library/Components/cart/cart.html:22]: Code Structure is good.
1. Bem Naming convention is easy to read.
2. Senmatic HTML well usd

**Comment by shruthipatil11** [comment]
Good to merge.

File: library/Components/cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    <section class="section">
+      <div class="card__quantity">
+        <div class="heading__container">
+          <span class="heading__cart">My Cart</span>
+        </div>
+        <div class="grid img__horizontal">
+          <img
+            src="/assets/images/wishlist_guitar1.jpg"
+            alt="Horizontal Image"
+          />
```
Comment (m26unkwn) [inline library/Components/cart/cart.html:59]: Space between product and checkout cart is too large.

File: library/Components/cart/cart.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        </div>
+      </nav>
+    </header>
+    <!-- NAV BAR END -->
+
+    <!-- CART SECTION START -->
+    <section class="section">
+      <div class="card__quantity">
+        <div class="heading__container">
+          <span class="heading__cart">My Cart</span>
```
Comment (m26unkwn) [inline library/Components/cart/cart.html:53]: You can center this my cart text horizontally.

**Review by m26unkwn — APPROVED** [review]
Good work UI is great.

**Review by shruthipatil11 — APPROVED** [review]

**Pull request merged**